SharedPackages/DataBrokerProtectionCore/Package.resolved

```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/content-scope-scripts",
       "state" : {
-        "revision" : "8fa69d009d3205b969bc76477897e3b82095f9f6",
-        "version" : "8.5.0"
+        "revision" : "313a212b112c8edaf4eccc0077400f26340efa6a",
+        "version" : "8.8.0"
```

Comment:
This should be excluded to keep the PR tight.

Reply:
Thanks, yeah this was pre-revert so this needs updating